### PR TITLE
fix the bug in latest on homepage

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@smn @bruskiza ready for review

File: tuneme/templates/core/tags/section_listing_homepage.html [hunk 1]
```diff
@@ -12,7 +12,23 @@
...
+      {% for article in sub_section.articles %}
+        {% if article.featured_in_latest %}
+          {% if article.image %}
+            <a href="{% pageurl article %}">{% image article.image width-60 %}</a>
+          {% endif %}
+          <h3>{{article.title}}</h3>
+          <p>{{article.subtitle}}</p>
+          {% get_comment_count for article as comment_count %}
+          <div class="comments-count"><a href="">{{ comment_count }} comment{{ comment_count|pluralize }}<span></span></a></div>
+          <a href="{% pageurl article %}" class="button">Read on</a>
```
Comment (smn) [inline tuneme/templates/core/tags/section_listing_homepage.html]: Please put "Read on" and "View Similar" in `{% trans ... %}` blocks so we can translate them at a later stage.

**Comment by smn** [comment]
:+1: once `{% trans ... %}` stuff is added.

**Comment by smn** [comment]
:+1:

**Comment by smn** [comment]
thanks!

**Pull request merged**